components/ws-proxy/pkg/proxy/workspacerouter.go

```diff
@@ -51,7 +52,14 @@ func HostBasedRouter(header, wsHostSuffix string) WorkspaceRouter {
 			}
 			blobserveRouter = r.MatcherFunc(matchBlobserveHostHeader(wsHostSuffix, getHostHeader)).Subrouter()
 			portRouter      = r.MatcherFunc(matchWorkspacePortHostHeader(wsHostSuffix, getHostHeader)).Subrouter()
-			ideRouter       = r.MatcherFunc(matchWorkspaceHostHeader(wsHostSuffix, getHostHeader)).Subrouter()
+
+			// Allow to match all cluster prefixes (ws-eu01, ws-eu01, ws-us01, ...)
+			// independent of the prefix of the cluster this component is
+			// deployed to. If the URL has an older WS cluster prefix that does
+			// not exist anymore, the workspaceMustExistHandler function in
+			// routes.go redirects to /start/.
+			allClusterWsHostSuffixRegex = fmt.Sprintf(`(\.ws[^\.]*)?\.%s`, strings.ReplaceAll(wsHostName, `.`, `\.`))
```

Comment:
We're still imposing this convention on workspace clusters. Is there a way around that (e.g. as discussed previously)?

Reply:
Moved that to the help chart and made it configurable in `values.yaml`.